### PR TITLE
interfaces/modem-manager: add access to PCIe modems

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**
Allow access to PCIe modems by using the modem-manager interfaces.

### CODE REVIEW

File: interfaces/builtin/modem_manager.go [hunk 1]
```diff
@@ -1218,12 +1221,28 @@ LABEL="mm_zte_port_types_end"
 # that don't have this tag.  MM will still get the udev 'add' event for the
 # device a short while later and then process it as normal.
 
-ACTION!="add|change|move", GOTO="mm_candidate_end"
+ACTION!="add|change|move|bind", GOTO="mm_candidate_end"
```
Comment (anonymouse64) [inline interfaces/builtin/modem_manager.go:1224]: do these changes come from an update to the modem-manager deb?

Comment (alfonsosanchezbeato) [reply interfaces/builtin/modem_manager.go:1224]: Yes, the udev rules changes come from the latest MM debian package.

**Review by anonymouse64 — APPROVED** [review]
lgtm

**Review by bboozzoo — APPROVED** [review]
LGTM

**Comment by alexmurray** [comment]
LGTM!

**Review by alexmurray — APPROVED** [review]

**Pull request merged**